Assignment0/matrixMatrixProduct.cpp

```diff
@@ -0,0 +1,32 @@
+#include "header.hpp"
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+std::vector<std::vector<double>> matrixMatrixProduct(const std::vector<std::vector<double>>& mat1, const std::vector<std::vector<double>>& mat2) {
+
+    size_t m = mat1.size();           // Rows in mat1
+    size_t n = mat1[0].size();        // Columns in mat1 
+    size_t q = mat2.size();           // Rows in mat2
+    size_t p = mat2[0].size();        // Columns in mat2
```

Comment:
It is preferred to have a meaningful name than a single-letter name for variables.